subjects/sum/README.md

```diff
@@ -0,0 +1,42 @@
+## sum
+
+### Instructions
+
+Write a function that takes two single digit numbers as a string and returns the sum as an int. If the number of digits from one of the arguments is more than one, return 0.
```

Comment:
Try this:

```Write a function that takes two strings as arguments. This function will have to return the sum of the two strings as an integer.

The strings can only contain one digit which will be a positive or negative number from 0 to 9. Otherwise, you must return 0.
```

Reply:
Thanks.